runtime/near-vm-runner/src/prepare.rs

```diff
@@ -11,14 +11,19 @@ pub(crate) const WASM_FEATURES: wasmparser::WasmFeatures = wasmparser::WasmFeatu
     // wasmer singlepass compiler requires multi_value return values to be disabled.
     multi_value: false,
     bulk_memory: false,
-    module_linking: false,
     simd: false,
     threads: false,
     tail_call: false,
     deterministic_only: false,
     multi_memory: false,
     exceptions: false,
     memory64: false,
+    component_model: false,
+    extended_const: false,
+    mutable_global: true,
+    relaxed_simd: false,
+    saturating_float_to_int: true,
```

Comment:
Similar comment as with the same change in wasmer PR – I believe these should be false, to avoid enabling new instructions we don't intend to support at this point in time. Especially because pwasm parser does not support them, but also enabling new instructions better be a conscious decision rather than an accidental one and keeping that in a separate PR would really help that.